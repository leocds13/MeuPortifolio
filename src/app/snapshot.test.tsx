import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Page from "./page";

it("renders a heading", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
