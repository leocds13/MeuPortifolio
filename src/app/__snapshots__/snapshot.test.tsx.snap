// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a heading 1`] = `
<div>
  <div>
    <h1>
      Home
    </h1>
    <a
      href="/about"
    >
      About
    </a>
  </div>
</div>
`;
